build_all.sh

```diff
@@ -3,11 +3,17 @@
 set -e
 
 if [[ "$1" == "-h" ]]; then
-    echo "build_all.sh [JetPack_version] [JetPack_Linux_source]"
+    echo "build_all.sh [--no-dev-dbg] [JetPack_version] [JetPack_Linux_source]"
```

Comment:
Did you try building it on Jenkins? JP5 + JP6?

Reply:
Yes, built and verified the output Images for JP5.0.2 and JP6. I'm able to enable dev_dbg() logs from d4xx.c file.